src/value_classes/value_id.rs

```diff
@@ -22,7 +22,8 @@ impl<'a> ValueList<'a> {
         let mut raw_string: *mut c_char = ptr::null_mut();
 
         let res = unsafe {
-            extern_manager::get_value_list_selection_as_string(manager_ptr, self.value_id.ptr, &mut raw_string, rust_string_creator)
+            let cvid = extern_value_id::value_id_from_packed_id(self.value_id.home_id, self.value_id.id);
+            extern_manager::get_value_list_selection_as_string(manager_ptr, &cvid, &mut raw_string, rust_string_creator)
         };
```

Comment:
same remark here: this seems more readable to have this conversion done in the C wrapper ?